src/Snackbar/snackbarStyles.ts

```diff
@@ -0,0 +1,60 @@
+import styled from '@emotion/styled';
+
+export interface SnackbarPropsStyles {
+    anchorOrigin: { vertical: string; horizontal: string };
+    open: boolean;
+}
+
+export const TRANSITION_CLOSE_TIME = 0;
+
+export const SnackbarWrapper = styled.div(() => ({
+    '.content-snackbar-wrapper': {
```

Comment:
As mentioned, I believe some extra styles are needed in the root of the `SnackbarWrapper` styles to properly position. At the very list,  I think we need the (something similar to) following:

```css
z-index: 1400; /* This should actually be theme.zIndex.snackbar */
position: fixed;
left: 24px;
bottom: 24px;
```

Not entirely sure if it needs to be added here or if it needs to be added in the `portal.style` updates, but I'd start here.